openqa-cli-completion.bash

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+_openqa_cli_completions() {
```

Comment:
How is this file installed? I would guess something needs to be added to the build and packaging?

Reply:
good point. we could add documentation. because what the user needs to do is to add it to the bashrc or zsh script and source it

Reply:
We could also create sub packages, like I did it here for git-subrepo:
https://build.opensuse.org/projects/devel:tools:scm/packages/git-subrepo/files/git-subrepo.spec?expand=1
So we would have `openqa-client-{bash,zsh}-completion`

Reply:
I dont know if I need to do anything in the spec file tho?! Can we have a chat tomorrow?

Reply:
Give it a try. Look what https://build.opensuse.org/projects/devel:tools:scm/packages/git-subrepo/files/git-subrepo.spec?expand=1 is doing for "bash-completion" and "zsh-completion" respectively. Then extend https://github.com/os-autoinst/openQA/blob/master/dist/rpm/openQA.spec by doing the equivalent.

Reply:
If it's in the spec, people only need to install one of the subpackages, rather than having to manually add something to their rc files

Reply:
Look at something like for example alacritty: https://build.opensuse.org/projects/openSUSE:Factory/packages/alacritty/files/alacritty.spec

There is a subpackage for the completion which supplements the main package and bash (so it is automatically installed if you have the main program and bash).
```
%package bash-completion
Summary:        Bash Completion for %{name}
Requires:       bash-completion
Supplements:    (%{name} and bash)
BuildArch:      noarch
```

and the subpackage just needs to drop the completion file into `%{buildroot}/%{_datadir}/bash-completion/completions/%{name}`